model_compression_toolkit/core/common/graph/virtual_activation_weights_node.py

```diff
@@ -183,3 +183,21 @@ def __init__(self,
                                          c.activation_quantization_cfg.activation_n_bits), reverse=True)
 
         self.candidates_quantization_cfg = v_candidates
+
+    def get_bops_count(self, fw_impl: Any, fw_info: FrameworkInfo, candidate_idx: int) -> float:
+        """
+        Computes the composed node's (edge) bit-operation count.
+
+        Args:
+            fw_impl: A FrameworkImplementation object with framework specific methods.
+            fw_info: A FrameworkInfo object with framework specific information,
+            candidate_idx: The index of the node's quantization candidate configuration.
+
+        Returns: The BOPS count of the composed node.
+
+        """
+        node_mac = fw_impl.get_node_mac_operations(self.original_weights_node, fw_info)
+        node_bops = self.candidates_quantization_cfg[candidate_idx].weights_quantization_cfg.weights_n_bits * \
+                    self.candidates_quantization_cfg[candidate_idx].activation_quantization_cfg.activation_n_bits * \
+                    node_mac
+        return node_bops
```

Comment:
Why not checking if quantization is enabled (if not use FLOAT_BITWIDTH)?

Reply:
Not sure to which part of the code this comment is referring